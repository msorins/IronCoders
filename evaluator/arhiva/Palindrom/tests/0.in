10
4 6 21 553 121 19 29 30 14 5