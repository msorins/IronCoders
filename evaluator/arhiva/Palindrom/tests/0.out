21 553 19 29 30 14